_layouts/default.html

```diff
@@ -29,6 +29,8 @@ <h1 class="site-name"><a href="{{ site.baseurl }}/">{{ site.name }}</a></h1>
           <nav>
             <a href="{{ site.baseurl }}/">Blog</a>
             <a href="{{ site.baseurl }}/about">About</a>
+            <a href="{{ site.baseurl }}/open-source">Open Source</a>
```

Comment:
@rx , since we currently don't have any specific things to highlight here, we can remove this link/page for for Open Source